app/services/dataworks_mappers/sdr.rb

```diff
@@ -46,420 +56,508 @@ def perform_map
     private
 
     # Not mapping alternative/subtitles at the moment
-    def titles(element = source.description.title)
-      return [] if element.blank?
+    def titles
+      return [] if (title = source.dig('description', 'title', 0, 'value')).blank?
```

Comment:
Will more complicated titles be encountered?

Reply:
They might be, but I can't bring in `TitleBuilder` anymore because we don't have `cocina-models`, and I wanted to defer re-implementing (ugh) its logic until I have a gem to hold that and other cocina-parsing responsibilities.